src/os/bluestore/BitAllocator.h

```diff
@@ -246,26 +246,20 @@ class BitMapArea {
 class BitMapAreaList {
 
 private:
-  BitMapArea **m_items;
-  int64_t m_num_items;
+  std::vector<BitMapArea*> m_items;
```

Comment:
I agree that using a straight std::vector (even without a typedef) is simpler and makes the code easier to understand.